### PR TITLE
Fix Jenkins credentials ID

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy [hunk 1]
```diff
@@ -19,7 +19,7 @@ pipeline {
 
         stage('Update ci.centos.org jobs') {
             steps {
-                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'jenkins-centos', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
+                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'centos-jenkins', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy]:
```suggestion
                withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
```

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy [hunk 2]
```diff
@@ -19,7 +19,7 @@ pipeline {
...
                     virtEnv('./ci', "cd ./centos.org && jenkins-jobs --conf ./centos_jenkins.ini --user '${env.USERNAME}' --password '${env.PASSWORD}' update -r ./jobs")
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy]:
```suggestion
                     virtEnv('./ci', "cd ./centos.org && jenkins-jobs --conf ./centos_jenkins.ini --user 'foreman' --password '${env.PASSWORD}' update -r ./jobs")
```

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-rpm.groovy [hunk 3]
```diff
@@ -35,7 +35,7 @@ pipeline {
 
                 git_clone_foreman_infra()
 
-                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'jenkins-centos', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
+                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'centos-jenkins', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-rpm.groovy]:
```suggestion
                withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
```

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-rpm.groovy]: and L43 needs `jenkins_username": "foreman",`

**Comment by ekohl** [comment]
Updated

**Pull request merged**